docs/server/source/server-reference/configuration.md

```diff
@@ -102,6 +110,18 @@ The settings with names of the form `database.*` are for the database backend
 * `database.replicaset` is only relevant if using MongoDB; it's the name of the MongoDB replica set, e.g. `bigchain-rs`.
 * `database.connection_timeout` is the maximum number of milliseconds that BigchainDB will wait before giving up on one attempt to connect to the database backend.
 * `database.max_tries` is the maximum number of times that BigchainDB will try to establish a connection with the database backend. If 0, then it will try forever.
+* `database.ssl` is a flag that determines if BigchainDB connects to the
+  backend database over SSL or not. This can be set to any value (the presence of a value
+  qualifies as set or true in this case) to enable SSL connections. Do not
+  specify this parameter to connect over plaintext connection.
```

Comment:
I was going through the code now, I was wrong about my understanding of the parsing process.
The `ssl` parameter can take the value `true` or `false`.
My statement is applicable when travis is run (or test are run locally).
I found a bug in my travis changes and have opened an issue for it. Will close it and then update the docs.